package ExceptionHandling;

import java.util.Scanner;

public class ExceptionDemo {
	public static void main(String[] args) {
		try {
			Scanner sc = new Scanner(System.in);
			System.out.println("enter i  = ");
			int i = sc.nextInt();
			System.out.println("entr j = ");
			int j = sc.nextInt();
			int k =i/j;
			System.out.println(k);
		} catch (Exception e) {
			
		}
	}
}
